deploy/examples/backup/OneTimeAwsBackup.yaml

```
@@ -0,0 +1,7 @@
+apiVersion: keycloak.org/v1alpha1
+kind: KeycloakBackup
+metadata:
+  name: example-keycloakbackup
+spec:
+  aws:
+    awsCredentialsSecretName: s3-backup
```

```suggestion
    credentialsSecretName: s3-backup
```
Since it is being split by provider (aws), this can be named generically for all.

---

Good point. Fixing.